test/BUILD

```diff
@@ -236,18 +236,18 @@ scala_library(
 #needed to test java sources are compiled
 scala_binary(
     name = "MixJavaScalaLibBinary",
-    srcs = ["src/main/scala/scala/test/MixJavaScalaLibBinary.scala"],
-    main_class = "scala.test.MixJavaScalaLibBinary",
+    srcs = ["src/main/scala/scalarules/test/MixJavaScalaLibBinary.scala"],
+    main_class = "scalarules.test.MixJavaScalaLibBinary",
     deps = ["MixJavaScalaLib"],
 )
 
 #Mix java scala (srcjar), much like above only the java is provided via srcjar
 scala_library(
     name = "MixJavaScalaSrcjarLib",
     srcs = glob([
-        "src/main/scala/scala/test/mix_java_scala/*.scala",
+        "src/main/scala/scalarules/test/mix_java_scala/*.scala",
         # srcjar created with `jar -cfM Baz.srcjar Baz.java`
```

Comment:
these comments were well with the extra few seconds it took to add them. I dealt with a number of failing tests related to jars that I had to recreate using the commentary in these rules.